content/tutorials/sweeps.md

```diff
@@ -6,219 +6,166 @@ menu:
 
-{{< img src="/images/tutorials/sweeps-1.png" alt="" >}}
-
-## Sweeps: An Overview
-
-Running a hyperparameter sweep with Weights & Biases is very easy. There are just 3 simple steps:
-
-1. **Define the sweep:** we do this by creating a dictionary or a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}) that specifies the parameters to search through, the search strategy, the optimization metric et all.
-
-2. **Initialize the sweep:** with one line of code we initialize the sweep and pass in the dictionary of sweep configurations:
-`sweep_id = wandb.sweep(sweep_config)`
-
-3. **Run the sweep agent:** also accomplished with one line of code, we call `wandb.agent()` and pass the `sweep_id` to run, along with a function that defines your model architecture and trains it:
-`wandb.agent(sweep_id, function=train)`
-
-
-## Before you get started
-
-Install W&B and import the W&B Python SDK into your notebook:
-
-1. Install with `!pip install`:
+## Before you start
 
+Install W&B and import the W&B Python SDK in your notebook:
 
+1. Install W&B:
 ```
 !pip install wandb -Uq
 ```
 
 2. Import W&B:
-
-
 ```
 import wandb
 ```
 
-3. Log in to W&B and provide your API key when prompted:
-
-
+3. Log in to W&B by entering your API key when prompted:
 ```
 wandb.login()
 ```
 
-## Step 1️: Define a sweep
-
-A W&B Sweep combines a strategy for trying numerous hyperparameter values with the code that evaluates them.
-Before you start a sweep, you must define your sweep strategy with a _sweep configuration_.
+## Step 1: Define a sweep
 
+A W&B Sweep combines a strategy for testing hyperparameter values with code that evaluates them. Define your sweep strategy with a _sweep configuration_ before starting the sweep.
 
 {{% alert %}}
-The sweep configuration you create for a sweep must be in a nested dictionary if you start a sweep in a Jupyter Notebook.
-
-If you run a sweep within the command line, you must specify your sweep config with a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}).
+For sweeps in a Jupyter Notebook, the sweep configuration must be in a nested dictionary. For command-line sweeps, use a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}).
 {{% /alert %}}
 
 ### Pick a search method
 
-First, specify a hyperparameter search method within your configuration dictionary. [There are three hyperparameter search strategies to choose from: grid, random, and Bayesian search]({{< relref "/guides/models/sweeps/define-sweep-configuration/sweep-config-keys/#method" >}}).
-
-For this tutorial, you will use a random search. Within your notebook, create a dictionary and specify `random` for the `method` key. 
-
+Specify a hyperparameter search method in your configuration dictionary. Choose from grid, random, and Bayesian search. Use random search in this tutorial. Create a dictionary in your notebook and set `method` to `random`.
```

Comment:
This list the "in this tutorial" clause and requires you to do some thinking about how you'd set it to one of the other modes (is it just lowercase `bayesian`?) Maybe it would be good to link to the API ref.